specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/ReplicationLinks.json

```diff
@@ -124,44 +124,6 @@
             "$ref": "./examples/ReplicationLinkGet.json"
           }
         }
-      },
-      "delete": {
-        "tags": [
```

Comment:
don't suggest to remove the operation if it is for fixing R4011- DeleteOperationResponses. you can add missing response code to fix it. removing a operation is serious breaking change, not only for SDK. let us know if there is other reason

Reply:
Removing the ReplicationLinks_Delete method from V20171001 is the decision from the owner @dlemMSFT and he has approved this PR. Please refer to this internal PR for the detail: https://msdata.visualstudio.com/Database%20Systems/_git/DsMainDev/pullrequest/496954?_a=files

There are two reasons we can safely remove this endpoint from the latest version:
1. there is one ReplicationLinks_Delete method in 2014-04-01 stable version, The SQL API router will auto re-direct the delete request to 2014-04-01 stable. At the same time, @dlemMSFT has confirmed this would not lead any issues. 
2. ReplicationLinks_Delete method from V20171001 was just introduced in Nov 2020. https://github.com/Azure/azure-rest-api-specs/search?q=ReplicationLinks_Delete . Via the search result, we know that the ReplicationLinks_Delete method from V20171001 has not been added to any SDK and doc. v4 SDK still uses the ReplicationLinks_Delete method from 2014-04-01 stable version.

Reply:
I see, but if we call ReplicationLinks_Delete via version 2020-08-01-preview. it will cause Swagger KPI completeness issue again as before. right?

Reply:
> if we call ReplicationLinks_Delete via version 2020-08-01-preview. it will cause Swagger KPI completeness issue again as before. right?
yes

Reply:
Hi @ericshape, will you revert this change? I mean it will cause KPI completeness again if you remove this operation

Reply:
yes. we will remove this operation. 'cause we have removed the impl of this API from SQL DB repo.